tools/nb_code_sync.py

```diff
@@ -123,11 +123,12 @@ def _is_source_code_equal(x_list, y_list):
 
   def _set_cell_source(self, cell_id, source):
     for i, cell in enumerate(self.data["cells"]):
-      if "id" not in cell["metadata"]:
+      try:
+        if cell["metadata"]["id"] == cell_id:
```

Comment:
I think we can revert the last commit and just change this to:
`if cell["metadata"].get("id") == cell_id:`
WIll that work?

Reply:
Sure, it works! :-D 👍